src/ParcelRegistry.Projections.Legacy/ParcelSyndication/ParcelSyndicationProjections.cs

```diff
@@ -125,18 +130,21 @@ await context
             When<Envelope<TerrainObjectWasImportedFromCrab>>(async (context, message, ct) => await DoNothing());
             When<Envelope<TerrainObjectHouseNumberWasImportedFromCrab>>(async (context, message, ct) => await DoNothing());
             When<Envelope<AddressSubaddressWasImportedFromCrab>>(async (context, message, ct) => await DoNothing());
+
             #endregion
 
             When<Envelope<ParcelWasMigrated>>(async (context, message, ct) =>
             {
+                var geometry  = new WKBReader().Read(message.Message.ExtendedWkbGeometry.ToByteArray());
+
                 var parcelSyndicationItem = new ParcelSyndicationItem
                 {
                     Position = message.Position,
                     ParcelId = message.Message.ParcelId,
                     CaPaKey = message.Message.CaPaKey,
                     Status = ParcelStatus.Parse(message.Message.ParcelStatus),
-                    XCoordinate = message.Message.XCoordinate,
-                    YCoordinate = message.Message.YCoordinate,
+                    XCoordinate = decimal.Parse(geometry.CentroidWithinArea().Coordinate.X.ToString()),
+                    YCoordinate = decimal.Parse(geometry.CentroidWithinArea().Coordinate.Y.ToString()),
```

Comment:
Create migration to remove XY coordinates and add geometry as extendedwkb (string = hex)